docs/how-to/deploy-cluster.md

```diff
@@ -0,0 +1,171 @@
+
+### Deployment and recommendations
+The next steps are described [here](https://github.com/input-output-hk/internal-documentation/wiki/Developer-clusters-HOWTO).
+Some notes and recommendations about this instruction:
+* `cardano-sl` git revision is most likely HEAD commit of your branch
+* Ask DevOps which `iohk-ops` branch you should use before deploy.
+* Specify `--bump-system-start-held-by` option for `io deploy`. 
+The passed value depends on how long will deployment take: 
+if `cardano-sl` is compiling from scratch, then it probably should be 15-20 mins, otherwise it's enough 5-10 mins. 
+If deploy won't be done utill system start then the first slots will be without blocks and system most probably will die.
+* Obtain logs from one of node after you deployed the system to make sure deployment was successful.
+
+## How to prepare installers
+
+Installer is file which can install Daedalus on your machine (`*.exe` file for Windows, `*.pkg` file for MacOS).
+
+### Which parts Daedalus consists of
+Before the process of preparing installers will be described I would like to say which parts Daedalus consists of.
+There are three parts: `launcher`, `cardano-node` and Daedalus frontend.
+`cardano-node` is like application which is launched on cluster, however, it also provides API to maintain your wallets, secret keys and etc.
+So essentially it's backend for Daedalus frontend.
+
+When you open Daedalus on your machine `launcher` is launched and it launches `cardano-node` and Daedalus frontend which uses provided API.
+So installer contains all these three parts.
+`cardano-node` comes from [cardano-sl repository](https://github.com/input-output-hk/cardano-sl), 
+`launcher` and Daedalus fronted come from [daedalus repository](https://github.com/input-output-hk/daedalus).
+
+### How to build installers
+Preparing installers implies that cluster which Daedalus will connect to is already deployed.
+Installers are a result of continuous integration build of daedalus repository commit. 
+As result of this build, we get an installer for Windows and for MacOS.
+
+#### Preparing confinguration.yaml
+As first step you should add to `configuration.yaml` in cardano-sl repository new configurations: one for Windows, one for MacOS.
+It should be derived from cluster configuration, for instance if you deployed `devnet` configuration these two configurations should be based
+on `devnet`.
+
+So these following configuration should be added for `devnet`:
+
+```
+devnet_staging_win64:
+  <<: *devnet
+  update:
+    <<:
+    applicationName: csl-daedalus
+    applicationVersion: 0
+    lastKnownBlockVersion:
+      bvMajor: 0
+      bvMinor: 0
+      bvAlt: 0
+    systemTag: win64
+
+devnet_staging_macos64:
+  <<: *devnet
+  update:
+    <<:
+    applicationName: csl-daedalus
+    applicationVersion: 0
+    lastKnownBlockVersion:
+      bvMajor: 0
+      bvMinor: 0
+      bvAlt: 0
+    systemTag: macos64
+```
```

Comment:
I didn't imagine that people would use `devnet` to test US, but it happens, so I propose to add these two configurations to `configuration.yaml` in `develop` (forever). Can you do it please?